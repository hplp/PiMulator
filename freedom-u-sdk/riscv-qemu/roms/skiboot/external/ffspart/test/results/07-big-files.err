Data file for partition 'ONE' is too large
