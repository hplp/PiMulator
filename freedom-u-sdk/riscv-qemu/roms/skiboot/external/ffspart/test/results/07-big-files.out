Adding 'ONE' 0x00000300, 0x00000100
Freeing hdr
